ibis/backends/risingwave/__init__.py

```diff
@@ -257,3 +346,311 @@ def _register_in_memory_table(self, op: ops.InMemoryTable) -> None:
             with self.begin() as cur:
                 cur.execute(create_stmt_sql)
                 extras.execute_batch(cur, sql, data, 128)
+
+    def exec_driver_sql(self, query: str):
```

Comment:
Why do you need this additional method? Try to avoid adding public APIs like this.

Reply:
Sorry, it should have been deleted. Removed just now.